### ISSUE TITLE
tensorflow/tensorflow

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#4152]
tensorflow/tensorflow<br>
Major Features And Improvements Update tf.keras to the Keras 2.1.6 API. tfe.Network is deprecated. Please inherit from tf.keras.Model. Adding support of core feature columns and losses to gradient boosted trees estimators. The distributions.<br>
https://ift.tt/2JjitHv
